webapp/.docker/docker-compose.yaml

```diff
@@ -0,0 +1,42 @@
+services:
```

Comment:
this file contains the directives that spin up the docker containers. See the README file for detailed explanation 
https://github.com/nsmq-ai/nsmqai/pull/7/files#diff-91584b86758870da3cf7b93afcb7ad605946bafc5f1cb0de288d16a9f55f8552R1